variables

```diff
@@ -53,6 +53,8 @@ export SUBMARINER_CHANNEL_RELEASE=""
 export SUBMARINER_IPSEC_NATT_PORT=4505
 export SUBMARINER_CABLE_DRIVER="libreswan"
 export SUBMARINER_GATEWAY_COUNT=1
+# The default catalog source is - redhat-operators
+export SUBMARINER_CATALOG_SOURCE ="submariner-catalog"
```

Comment:
If you are explicitly setting the downstream submariner catalog as a global variable, you need to update the reference to it in all other places.

Also, I would recommend to rename the variable to - `SUBMARINER_DOWNSTREAM_CATALOG_SOURCE`.
Because if we are making an upstream deployment, the original `redhat-marketplace` catalog should be used.